main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -275,6 +275,8 @@ function registerImporting() {
   IM.registerMimeType("text/n3", "text/rdf/n3");
   IM.registerMimeType("text/rdf+n3", "text/rdf/n3");
   IM.registerMimeType("text/turtle", "text/rdf/ttl");
+  IM.registerMimeType("application/xml", "text/xml");
+  IM.registerMimeType("text/xml", "text/xml");
```

Comment:
It would be good to also recognize the `+xml` suffix as a fallback when we don't have a more specific type, e.g. `application/atom+xml` but that would require introducing additional generality into the underlying code, so perhaps we can spin that off as a separate issue. We already support a few of the more specific ones such as `application/rdf+xml`.